app/models/user.rb

```diff
@@ -526,6 +532,18 @@ def following?(other_user)
     following.include?(other_user)
   end
 
+  def auto_watching?(other_user)
+    following?(other_user) ? Following.find_by(follower_id: self, followed_id: other_user).watch : false
```

Comment:
【IMO】
booleanのカラムがあると `#{カラム名}?` というメソッドが生えるので、以下のように書き換えられます
外から見てbooleanを返すのが直感的に分かるので、自分はこちらのほうが好みです

```suggestion
    following?(other_user) ? Following.find_by(follower_id: self, followed_id: other_user).watch? : false
```

Reply:
確かにこちらの方が`watch`という属性がbooleanであることが分かりやすいですね。
修正しました。